samples/core/console-apps/Fibonacci/README.md

```diff
@@ -1,7 +1,7 @@
 Fibonacci Sample
 ================
 
-This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/articles/core/tutorials/using-with-xplat-cli)
+This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/core/tutorials/using-with-xplat-cli)
```

Comment:
@BillWagner these are supposed to be the readme files for the samples. they are not published so if you're looking at this file on your machine, you need the absolute URL.